webgl/idlharness.window.js

```diff
@@ -13,3 +13,12 @@ idl_test(
   },
   'webgl1 interfaces'
 );
+
+idl_test(
+  ['webgl2'],
```

Comment:
Maybe you also tried this, but I put `['webgl1', 'webgl2']` as the first argument here, since it seems like exactly the use case for more than one source, but that causes "20 duplicate test names", seemingly all involving "WebGL2RenderingContext" even though that interface isn't in webgl1.idl. Simply concatenating the two IDL files into one doesn't change matters, so it's not because of something `idl_test` does wrong.

Can you make sense of the problem?

Note that when using two `idl_test`s like this, the subtests are interleaved, in particular the first two are "webgl1 interfaces" and "webgl2 interfaces". Trying to work out which IDL file a problem is from might be tricky. So maybe two separate test files would be better?